website/source/docs/providers/google/r/compute_health_check.html.markdown

```diff
@@ -0,0 +1,128 @@
+---
+layout: "google"
+page_title: "Google: google_compute_health_check"
+sidebar_current: "docs-google-compute-health-check"
+description: |-
+  Manages a Health Check within GCE.
+---
+
+# google\_compute\_health\_check
+
+Manages a health check within GCE. This is used to monitor instances
+behind load balancers. Timeouts or HTTP errors cause the instance to be
+removed from the pool. For more information, see [the official
+documentation](https://cloud.google.com/compute/docs/load-balancing/health-checks)
+and
+[API](https://cloud.google.com/compute/docs/reference/latest/healthChecks).
+
+## Example Usage
+
+```js
```

Comment:
You can use `tf` here (I was surprised, too!):

```tf
resource "google_compute_health_check" "default" {
  name = "test"
}
```

I won't comment on all the others, but they can all be updated :)